### ISSUE TITLE
ArcticDB only works when I specify symbol

### ISSUE THREAD

**Issue by CadePGCM** [man-group/dtale#789]
```
uri = myAmazonS3uri
global_state.use_arcticdb_store(uri=uri, use_store= True)
app = build_app(reaper_on=False)

if __name__ == '__main__':
    app.run(host="0.0.0.0", port=9207)
```
    
 This fails to load my data, as does not using the use_store parameter. Similarly fails when I use .show(). The documentation suggests this should work.

However, if I specify the symbol and library, ie. with:
`dtale.show_arcticdb(uri=uri, library='min_data', symbol='AACT_full_1min_adjsplitdiv')`
Then it works as intended.

**Comment by aschonfeld** [comment]
So I believe your issue is that you have not specified a library when using `use_arcticdb_store` so if you switch your code to this it should work:

```
uri = myAmazonS3uri
global_state.use_arcticdb_store(uri=uri, library='min_data')

if __name__ == '__main__':
      dtale.show(data='AACT_full_1min_adjsplitdiv', subprocess=False, reaper_on=False, host="0.0.0.0", port=9207)
```

Let me know how it goes.  Good luck

**Comment by CadePGCM** [comment]
Your example runs but doesn't address the issue, because you are specifying the symbol.

In the documentation, it suggests you can load the database without specifying the symbol and then use the UI to view and access different symbols. Adding the library (and not the symbol) has not addressed the issue.

Here is the specific error occurs when I attempt to load any symbol with the UI (having not pre-specified the symbol):

`read_descriptor(): incompatible function arguments. The following argument types are supported: 1. (self: arcticdb_ext.version_store.PythonVersionStore, arg0: Union[int, str], arg1: arcticdb_ext.version_store.PythonVersionStoreVersionQuery, arg2: arcticdb_ext.version_store.PythonVersionStoreReadOptions) -> arcticdb_ext.version_store.DescriptorItem Invoked with: <arcticdb_ext.version_store.PythonVersionStore object at 0x000001FAC7578670>, 'AACG_full_1min_adjsplitdiv', <arcticdb_ext.version_store.PythonVersionStoreVersionQuery object at 0x000001FAC7578B70>
Traceback (most recent call last):
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\views.py", line 119, in _handle_exceptions
    return func(*args, **kwargs)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\views.py", line 4300, in load_arcticdb_symbol
    startup(data=data_id)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\views.py", line 994, in startup
    global_state.new_data_inst(data_id)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\global_state.py", line 388, in new_data_inst
    new_data = instance or self._data_store.build_instance(data_id)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\global_state.py", line 279, in build_instance
    return DtaleArcticDBInstance(data, data_id, self)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\dtale\global_state.py", line 163, in __init__
    self._rows = lib._nvs.get_num_rows(self.symbol)
  File "C:\Users\Passage11\anaconda3\envs\vbtpro\lib\site-packages\arcticdb\version_store\_store.py", line 2320, in get_num_rows
    dit = self.version_store.read_descriptor(symbol, version_query)
TypeError: read_descriptor(): incompatible function arguments. The following argument types are supported:
    1. (self: arcticdb_ext.version_store.PythonVersionStore, arg0: Union[int, str], arg1: arcticdb_ext.version_store.PythonVersionStoreVersionQuery, arg2: arcticdb_ext.version_store.PythonVersionStoreReadOptions) -> arcticdb_ext.version_store.DescriptorItem

Invoked with: <arcticdb_ext.version_store.PythonVersionStore object at 0x000001FAC7578670>, 'AACG_full_1min_adjsplitdiv', <arcticdb_ext.version_store.PythonVersionStoreVersionQuery object at 0x000001FAC7578B70>`

Additionally only a handful of symbols in the library appear present in the UI, if that is relevant as well.

**Comment by aschonfeld** [comment]
@CadePGCM can you tell me which version of arcticdb you're using?  There has been a lot of issues with v1.6.1 of arcticdb.  I haven't had a chance to publish a new version of D-Tale blocking that version

**Comment by aschonfeld** [comment]
@CadePGCM I've fixed this issue with ArcticDB in v3.4.0 of D-Tale.  It's on pypi and should be on conda-forge shortly.  Let me know if you have any issues.  

Also, if you haven't already, please put your ⭐ on the repo when you get a sec. Thanks! 🙏
